tests/test_units.py

```diff
@@ -70,6 +71,11 @@ def test_fraction(self):
         out = convert_units_to(xr.DataArray([10], attrs={"units": "%"}), "")
         assert out == 0.1
 
+    def test_lazy(self, pr_series):
```

Comment:
I thought this was covered in test_indicators.test_delayed.

Reply:
There is no units conversion in that test! In fact, most calls to `convert_units_to` in `xclim.indices` only act on thresholds and stuff like that, which does not impact the laziness. And the indices where the dataarray was converted were not tested for laziness.

Reply:
Ok thanks for catching this !